src/nsp/HISTORY.rst

```diff
@@ -4,6 +4,13 @@ Release History
 ===============
 ===============
 
+##### 0.3.1
```

Comment:
version should be updated in setup.py and follow cli extension version schema here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md